pvlib/tests/iotools/test_solcast.py

```diff
@@ -118,5 +119,97 @@ def test_get_solcast_tmy(
     )
 ])
 def test_solcast2pvlib(in_df, out_df):
-    df = solcast2pvlib(in_df)
+    df = _solcast2pvlib(in_df)
     pd.testing.assert_frame_equal(df.astype(float), out_df.astype(float))
+
+
+@pytest.mark.parametrize("endpoint,function,params,json_response", [

```

Comment:
Why bother with the parametrization boilerplate if you are only testing one case?